sql3/planner/inbuiltfunctionsdate.go

```diff
@@ -150,6 +166,34 @@ func (n *callPlanExpression) EvaluateDatepart(currentRow []interface{}) (interfa
 
 }
 
+// EvaluateDateTimeFromParts evaluates the call to date_time_from_parts. This uses the base time.Date() function.
+func (n *callPlanExpression) EvaluateDateTimeFromParts(currentRow []interface{}) (interface{}, error) {
+	timestamps := make([]int, len(n.args))
```

Comment:
I see we always want to create date and time. A date only variation accepting just YYYY, MM, DD would be nice to have.

Reply:
thats true. We can add more going forward.